deploy/addons/heapster-rc.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    kubernetes.io/cluster-service: 'true'
+    k8s-app: heapster
+    name: heapster
+  name: heapster
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    kubernetes.io/cluster-service: "true"
+    k8s-app: heapster
+  template:
+    metadata:
+      labels:
+        kubernetes.io/cluster-service: 'true'
+        k8s-app: heapster
+    spec:
+      containers:
+      - name: heapster
+        image: kubernetes/heapster:canary
```

Comment:
Sure, I was just following what official heapster repo did.  I can change to 0.19